meta-zarhus-bsp-rockchip/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -1,6 +1,19 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += "file://0001-orangepi-cm4-base-rk3566_defconfig-copy-from-linux-t.patch"
+
+SRC_URI += "file://enable-optee.cfg"
+
+# SRC_URI += "file://enable-debug-log.cfg"
+
+# TODO:
+# modify the rkbin recipe to create tee-rk3566.elf first !!!
```

Comment:
@macpijan I though that rather than patching U-Boot, we wrap the bin into elf.

I have used the 

```
ENTRY(_binary_tee_rk3566_bin_start);

SECTIONS
{
        . = 0x08400000;
        .data : {
                *(.data)
        }
}
```

and 

```
aarch64-linux-gnu-objcopy -B aarch64 -I binary -O elf64-littleaarch64 tee-rk3566.bin bl32.o
aarch64-linux-gnu-ld bl32.o -T tee.ld -o tee-rk3566.elf
```

That can be further generalized and pushed into usptream rkbin recipe later on as well.

That worked best for me when integrating this, but this was a single step done manually.